### ISSUE TITLE
Identify unique data properties

### ISSUE THREAD

**Issue by mwthink** [mwthink/denton-jailbot#1]
Right now, the app will fetch the current list of inmates and then saves them to the database. It will gladly save duplicate data and there is no understood method to track updates vs new records. 

There are a few properties on the returned data that have unknown meaning/format, namely `bookhandle` and `bookno`. After a few runs of the bot, we should make an attempt to know which properties (if any, or both) might be a reliable unique identifier. 

Ideally, we'd want to be able to know what is a new arrest vs an updated arrest. This could help us identify when we need to update an existing record (e.g. The record initially did not have a mugshot or certain charges, but now it does) vs when we need a new record for an existing person (e.g. An inmate was in the jail, then released, then arrested again on the same day).

If we can determine a reliable unique identifier for an _InmateRecord_, then we should be able to answer these questions pretty easily. 

Research on the topic should be posted here. The inline JSdocs should also be updated to reflect this information once it is known.

**Comment by mwthink** [comment]
After a day of data scrapes, I've identified a few patterns that lead me to think these are true:

1. Both `bookhandle` and `bookno` are a consistent identifier for the duration of the inmate stay
2. Both values resemble an auto-incrementing value, although I seem to have some gaps in the counts. Even with gaps, the offset is the same for both values (.e.g `2544 = 20002031` and `2549 = 20002036`)

I have also noticed these (but am not sure if they are _always_ true):
1. The `datetimebooked` and `name` values never change during the stay
2. Most other values (outside of the book IDs) can and do change
3. Some records have mugshots from their very first appearance in the system, others have them added later.

I think it's sane to use `bookhandle` and/or `bookno` as consistent identifiers. They could be used to calculate a hash that serves as a persistent "stay ID" value.
